scripts/src/java/org/oppia/android/scripts/testing/TestBazelWorkspace.kt

```diff
@@ -246,7 +246,7 @@ class TestBazelWorkspace(private val temporaryRootFolder: TemporaryFolder) {
     if (!isConfiguredForKotlin) {
       // Add support for Kotlin: https://github.com/bazelbuild/rules_kotlin.
       val rulesKotlinReleaseUrl =
-        "https://github.com/bazelbuild/rules_kotlin/releases/download/v1.5.0-beta-3" +
+        "https://github.com/bazelbuild/rules_kotlin/releases/download/v1.7.1" +
```

Comment:
Is this a test-specific upgrade?

Reply:
Yeah. A lot of this code was originally written in #4937 and haphazardly copied to this PR, and that PR's code originated in #4886. The Bazel PR updates rules_kotlin to 1.7.1 and I figured we can just keep that bit here even though it's not in exact alignment for a few releases. It doesn't make much of a difference in the build graph between tests that use this utility and production.

Reply:
Sounds good.